src/Uno.UWP/Security.Credentials/PasswordVault.Android.cs

```diff
@@ -137,7 +190,7 @@ protected override void Dispose(bool disposing)
 					}
 					_isDisposed = true;
 
-					if (disposing)
+					if (disposing && (Android.OS.Build.VERSION.SdkInt > BuildVersionCodes.LollipopMr1))
```

Comment:
In all case, a cipher output stream is created, but it not closed for `<=lollipop`? Are there any particular reason for that?

Reply:
I forgot to annotate this section.
It cause a instance crash when disposing instantly for low level API.
When analyzing the state, it look like already dispose. I'm not sure about the exact behavior

Reply:
update:
The crash was due to: https://stackoverflow.com/questions/38543631/how-to-resolve-input-must-be-under-256-bytes-in-android
See also: https://github.com/xamarin/Essentials/blob/1.3.1/Xamarin.Essentials/SecureStorage/SecureStorage.android.cs#L121